documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -551,36 +889,46 @@ The unwrapping of the intensity of the spin-wave located around :math:`[0,-1,1]`
    :width: 500px
    :alt: Q-phi cut
 
-   Spin-wave scattering intensity around :math:`[0,-1,1]` point, expressed in spherical coordinate system.
-   
-Visible gap caused by missing detectors is obvious in :math:`[-50^o:+50^o]` angles range. Averaging over all 
-:math:`\theta` angles substantially improves statistics:
+   MAPS Fe data; Spin-wave scattering intensity the the origin centred
+   about the :math:`[0,-1,1]` Bragg peak. A visible gap caused by
+   missing detectors is obvious in the :math:`\phi`-axis range
+   :math:`[-50^\circ:+50^\circ]`.
+   Inset: Linear projection of the same region, the red lines show the
+   approximate mapping from the linear to spherical projections.
+
+Integrating over all :math:`\theta` and thus including other detectors
+substantially improves statistics, this is done by setting the
+:math:`\theta` parameter to ``[0, 180]``:
 
 .. code-block:: matlab
 
-    s2    = cut(data_source,sp_proj,[0,0.1,2],[0,180],[-180,4,180],[50,60]);
+    s2 = cut(data_source, sp_proj, [0, 0.1, 2], [0, 180], [-180, 4, 180], [50, 60]);
 
 .. figure:: ../images/spin_w_theta_av.png
    :align: center
    :width: 500px
    :alt: Q-phi cut theta-averages
 
-   :math:`\theta`-averaged spin-wave scattering intensity around :math:`[0,-1,1]` point.
-   
-and finally, 1D cut provides the intensity distribution as function of |Q|-distance from the spin-wave centre:
+   MAPS Fe data; Scattering intensity from cut averaged over all :math:`\theta`
+   spin-wave with the origin centred at the :math:`[0,-1,1]` Bragg
+   peak.
+
+The 1D cut below, generated by integrating over the :math:`\phi`-axis,
```

Comment:
and theta, yes?

Reply:
Yes and a subsection of energy, but this is a series of operations and the previous plot was already integrated over all theta.